.github/workflows/no-response.yml

```diff
@@ -0,0 +1,26 @@
+name: No Response
+
+## Closes issues marked with 'more information required' label after 7 days
+##    if original poster does not respond within that time
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
```

Comment:
I assume this is already being used somewhere in SF land?

Reply:
Nope :) CLI is still using the Probot App that's having issues. We are pinned to a specific version here though. I could go the extra step and pin to the commit hash.